src/GafferSceneUI/AttributeInspector.cpp

```diff
@@ -248,18 +248,36 @@ IECore::ConstObjectPtr AttributeInspector::value( const GafferScene::SceneAlgo::
 	return nullptr;
 }
 
-IECore::ConstObjectPtr AttributeInspector::fallbackValue( const GafferScene::SceneAlgo::History *history ) const
+IECore::ConstObjectPtr AttributeInspector::fallbackValue( const GafferScene::SceneAlgo::History *history, std::string &description ) const
 {
 	if( const auto inheritedAttribute = history->scene->fullAttributes( history->context->get<ScenePlug::ScenePath>( ScenePlug::scenePathContextName ) )->member( m_attribute ) )
```

Comment:
This initial call to `fullAttributes()` is doing all the work we're doing below, and more. Can we omit it?

Reply:
Omitted in 28a8397.